.travis.yml

```diff
@@ -49,6 +48,7 @@ deploy:
   email: ${NPM_USER_EMAIL}
   api_key: ${NPM_USER_API_KEY}
   on:
+    condition: '$ASCIIDOCTOR_CORE_VERSION = $ASCIIDOCTOR_CORE_VERSION_PUBLISH'
```

Comment:
I think you can use `branch` instead on `condition`.

```
on:
    branch: master
```

https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A

Reply:
forget that I misunderstood